CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
   ([#7888](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7888))
 - Set up EarlyInitAgentConfig even earlier
   ([#8413](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8413))
+- Add runtime-metrics-java17 ([#8165](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8165) and [#7886](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7886))
```

Comment:
This should go into the `Unreleased` section

Reply:
Do you mean under the "Unreleased" header? I think it's already within the unreleased section.

Reply:
Currently it looks like it's under `## Version 1.26.0 (2023-05-11)`

We don't require PRs in this repo to add change log entries (so people usually don't), and instead we generate the draft change log before making each release based on the PR titles, and then do bit of manual organization/clean up of that draft.

Reply:
Oh yes, you're right. Ok, I'll just remove the changelog entry